UACloudLibraryServer/AWSFileStorage.cs

```diff
@@ -51,13 +49,18 @@ public class AWSFileStorage : IFileStorage
     {
         private readonly string _bucket;
         private readonly string _prefix;
-        private readonly RegionEndpoint _region;
+      
+        private readonly IAmazonS3 _s3Client;
+        private readonly ILogger _logger;
+      
         /// <summary>
         /// Default constructor
         /// </summary>
-        public AWSFileStorage()
+        public AWSFileStorage(IAmazonS3 s3Client, IConfiguration config, ILoggerFactory logger)
         {
-            var connStr = Environment.GetEnvironmentVariable("BlobStorageConnectionString");
+            _s3Client = s3Client;
+            _logger = logger.CreateLogger("AWSFileStorage");

```

Comment:
Please also add:
loggerFactory.AddConsole(Configuration.GetSection("Logging")); 
loggerFactory.AddDebug();
in startup.configure method.

Reply:
I see the current code has services.AddLogging(builder -> builder.AddConsole()), which IMO is sufficient/better approach.   I can remove the 'AddConsole' call as that's not necessary when default builder is used.    BTW the syntax you had above is obsolete in 3+.

Reply:
Sorry, I missed that, then we're good, thanks so much for your contribution!